vars/piperExecuteBin.groovy

```diff
@@ -25,6 +26,23 @@ void call(Map parameters = [:], stepName, metadataFile, List credentialInfo, fai
 
         writeFile(file: ".pipeline/tmp/${metadataFile}", text: libraryResource(metadataFile))
 
+        // The default pipeline config, plus any custom default configs, have been
+        // extracted from the library resources into .pipeline/ by setupCommonPipelineEnvironment.groovy
+        // and fed into the DefaultValueCache. The 'default_pipeline_environment.yml' file
```

Comment:
would it make sense to also process 'default_pipeline_environment.yml' in setupCommonPipelineEnvironment?

Reply:
It already happens, but via prepareDefaults() and then in DefaultValueCache. It is not well visible from the code. But I've removed this, since Daniel's remarks about the defaults being in the go metadata has convinced me.